01.general/18.stealth-mode/docs.en.md

```diff
@@ -68,24 +68,24 @@ When the browser addresses a page, the server assigns it an ETag, which the brow
 
 **Block WebRTC**
 
-WebRTC (Real Time Communications) is a technology that allows streaming of data directly between browsers and apps. It can let others know your real IP address even if you use a proxy or VPN.
+WebRTC (Real Time Communications) is a technology allows the streaming of data directly between browsers and apps. It can let others know your real IP address, even if you use a proxy or VPN.
 Enabling this option can disrupt the work of certain browser applications, such as messengers, chats, cinemas, or games.
 
 **Block Push API**
 
-Push API enables servers to send messages to web applications regardless of browser status. This means you may see notifications from varous websites even if your browser is hidden to tray or not launched. Enable this option to block browser Push API completely.
+The Push API enables servers to send messages to web applications regardless of browser status. This means you may see notifications from various websites even if your browser is hidden in the tray or not launched. Enable this option to block the browser's Push API completely.
 
 **Block Location API**
 
-Enabling this option will prohibit the browser from sending GPS data that could be used to find your location and modify your search results or otherwise influence your web experience.
+Enabling this option will prevent the browser from sending GPS data that could be used to find your location, modify your search results, or otherwise influence your web experience.
```

Comment:
```suggestion
Enabling this option will prevent the browser from sending GPS data that could be used to determine your location, modify your search results, or otherwise influence your web experience.
```